doctr/io/elements.py

```diff
@@ -311,65 +316,66 @@ def export_as_xml(self, file_title: str = "docTR - XML export (hOCR)") -> Tuple[
             },
         )
         # iterate over the blocks / lines / words and create the XML elements in body line by line with the attributes
-        for block in self.blocks:
```

Comment:
It is a while ago i wrote this :sweat_smile: but we should test that it still works as expected with the PDF/A notebook

Reply:
i looked at it a little bit. in google colab the installation using git doesn't seem to work for me. However, the code should work just the output will be slightly different i think.